test/controllers/users.js

```diff
@@ -226,6 +242,50 @@ describe('users controller', () => {
       });
     });
   });
-
+  describe('/admin/users POST', () => {
+    it('should fail because the user is not an admin', (done) => {
+      successfulLoginNotAdmin().then((res) => {
+        chai.request(server)
+          .post('/admin/users/')
+          .set(sessionManager.HEADER_NAME, res.headers.authorization)
+          .send({
+            firstName: 'xxxxxx', lastName: 'zzzzzzzzz',
+            password: '1234', email: '123456@wolox.com.ar'
+          })
+          .catch((err) => {
+            err.response.should.be.json;
+            err.response.body.should.have.property('error');
+            err.response.body.error.should.equal('User is not allowed');
+            err.should.have.status(401);
+          })
+          .then((response) => {
+            done();
+          });
+      }).catch((err) => {
+        done(err);
+      });
+    });
+    it('should be successful creating a new admin', (done) => {
+      successfulLogin().then((res) => {
+        chai.request(server)
+          .post('/admin/users/')
+          .set(sessionManager.HEADER_NAME, res.headers.authorization)
+          .send({
+            firstName: 'xxxxxx', lastName: 'zzzzzzzzz',
+            password: '12345678910', email: '123456@wolox.com.ar'
+          })
+          .then((response) => {
+            response.should.have.status(201);
```

Comment:
We could also check that there is a new user created with the right attributes and the attribute `isAdmin` setted in true.